docs/maps/search.asciidoc

```diff
@@ -13,13 +14,51 @@ You can create a layer that requests data from {es} from the following:
 
 ** Grid aggregation source
 
-** <<terms-join>>
+** <<terms-join>>. The search context is applied to both the terms join and the vector source when the vector source is provided by Elasticsearch documents.
 
 * <<heatmap-layer>> with Grid aggregation source
 
 [role="screenshot"]
 image::maps/images/global_search_bar.png[]
 
+[role="xpack"]
+[[maps-create-filter-from-map]]
+=== Creating filters from your map
+
+You can create two types of filters by interacting with your map:
+
+* <<maps-spatial-filters, Spatial filters>>
+* <<maps-phrase-filter, Phrase filters>>
+
+[float]
+[[maps-spatial-filters]]
+==== Spatial filters
+
+Spatial filters narrow search results to documents that either intersect, are within, or do not intersect with the specified geometry.
+
+You can create spatial filters in two ways:
+
+* Clicking image:maps/images/tools_icon.png[] to define the spatial fitler by drawing a polygon or a bounding box on the map.
```

Comment:
* Click the tool icon image:maps/images/tools_icon.png[], and then draw a polygon or bounding box on the map to define the spatial filter.